### ISSUE TITLE
Converting JSON to Tableau ready (on OSX)

### ISSUE THREAD

**Issue by karisantos** [bigdatahci2015/forum#19]
Hello - I've spent almost an hour looking around the web for how to convert the JSON file to something Tableau can use.  This seems quite challenging to me, especially on  a Mac.  I am considering Apache Drill, but could use a hint or some reassurance. I see conflicting info on what works on Tableau Desktop for Mac - I think Tableau Data Extract API does not. And I'm not sure if Tableau ODBC works on Mac.

Anyone care to share a couple hints or useful sites?  Anyone do this on a Mac yet?

Thanks, Kari

**Comment by zhya215** [comment]
Google 'convert json into csv'
You will find what you want.

**Comment by bdmckean** [comment]
Hi Kari,
I have a Mac and adapted a python script that I found on the web for converting JSON to CSV.
-Brian

-----Original Message-----

**Comment by kjblakemore** [comment]
I cut/paste the tables from the markdown document to a txt file, then import the txt file into tableau.

**Comment by karisantos** [comment]
Thanks for all the help.  I tried https://www.npmjs.com/package/json-2-csv , which works, except it doesn't handle the Instructors array, leaving it just Object.  Has anyone found a converter that can handle the "Instructors" array?

**Comment by TheDataLeek** [comment]
I just finished a custom python program that converts the json structure into a sqlite relational database. @doubleshow , am I allowed to post this?

**Comment by doubleshow** [comment]
yes. you are allowed to post the script. thanks. but please don't share the resulting output as your classmates who choose to use your script need to figure out how to run the script as a part of the learning exercise.

**Comment by TheDataLeek** [comment]
Here is my script.

https://gist.github.com/willzfarmer/f185c25c872a221cc3ca
